source/embedded-shell.txt

```diff
@@ -20,8 +20,12 @@ to test queries and operations in your database.
 Open the Embedded MongoDB Shell
 -------------------------------
 
-To open the embedded :binary:`mongosh`, click :guilabel:`_MONGOSH`
-at the bottom of the |compass| window.
+To open the embedded :binary:`mongosh`,you can either:
+
+- Click :guilabel:`>_` on the right of your deployment name in the
+  :guilabel:`Connections Sidebar`. 
+- Click :guilabel:`>_Open MongoDB Shell` in the top right of the
+  :guilabel:`Databases` tab. 
```

Comment:
You can actually click this button in any connection-specific tab, and the button has "shell" written in lowercase. Thanks Lauren!

Reply:
Fixed!

Reply:
I'm not sure if "connection window" is the best way to describe this location since an application window is different from a tab. What would you think of rewording this bullet to the following? 

> Click `>_Open MongoDB shell` in the top right of any tab connected to a MongoDB deployment.